python/modules/task.py

```diff
@@ -313,105 +376,78 @@ def execute_specify(self, dict_local):
-        filename = dict_local["filename"]
-        output_path = FM.path_join(path, filename)
-        dict_key_name = dict_local["dict_key_name"]
-        data = dict_local[dict_key_name]
-        try:
-            delimiter = dict_local["delimiter"]
-        except:
-            delimiter = ","
-        R_connection.write_dataframe_to_csv(output_path, data, delimiter)
+    def execute_specify(self, dict_local, dict_setts):
+        output_path = FM.path_join(dict_setts["output_path"], dict_setts["filename"])
+        data = dict_local[dict_setts["dict_key_name"]]
+        R_connection.write_dataframe_to_csv(output_path, data, dict_setts["delimiter"])
 
 class TASK_MERGE_CSV(TASK):
-  
-    def __init__(self, parameters_by_value, parameters_by_name, updates_by_value, updates_by_name, args = {}):
-        TASK.__init__(self, parameters_by_value, parameters_by_name, updates_by_value, updates_by_name, args)
 
-    def execute_specify(self, dict_local):
-        in_path_list = []
-        in_path_list.append(dict_local["input_path_1"])
-        in_path_list.append(dict_local["input_path_2"])
-        csv_names = (dict_local["csv_names_list"]).split(",")
-        output_path = dict_local["output_path"]
-        try:
-            delimiter = dict_local["delimiter"]
-        except:
-            delimiter = "," #choose separator
-        for csv_name in csv_names:
-            data = CSV_M.simple_merge(csv_name, in_path_list, delimiter)
-            #Saving data
-            out_path = FM.path_join(output_path, csv_name)
-            if FM.path_check_existence(out_path):
-                logger.warning("File %s already exists. Removing old data.", out_path)
-                FM.dir_remove(out_path)
-            CSV_M.write_csv(out_path, delimiter, data) #if we would like to write_csv somewhere...
-            # PLACEHOLDER adding data to dictionary 
+    dict_task = {"input_path_1" : {"required" : True},
+                 "input_path_1" : {"required" : True},
+                 "csv_names_list" : {"required" : True},
+                 "output_path" : {"required" : True},
+                 "delimiter" : {"required" : True, "default" : ","}}
 
-class TASK_COPY_CSV(TASK):
-  
     def __init__(self, parameters_by_value, parameters_by_name, updates_by_value, updates_by_name, args = {}):
         TASK.__init__(self, parameters_by_value, parameters_by_name, updates_by_value, updates_by_name, args)
 
-    def execute_specify(self, dict_local):
+    def execute_specify(self, dict_local, dict_setts):
         in_path_list = []
-        in_path_list.append(dict_local["input_path"])
-        csv_names = (dict_local["csv_names_list"]).split(",")
-        output_path = dict_local["output_path"]
-        try:
-            delimiter = dict_local["delimiter"]
-        except:
-            delimiter = "," #choose separator
+        in_path_list.append(dict_setts["input_path_1"])
+        in_path_list.append(dict_setts["input_path_2"])

```

Comment:
Well, it's a very very old task that I didn't want to edit before today's meeting (I just replaced dict_local with dict_setts), because I wanted to make proposal to remove it (OR, optionally, replace with merge_list_csv). I just forgot to mention it on my post "in additional" section.
It was created to allow us to merge 2 chosen csv (i.e. during 2 parts experiments), but in fact we're using task_merge_subdir_csv to meet that purpose.